### PR TITLE
UIIN-79: Remove results display by default

### PR DESCRIPTION
**Pull request by YevheniiMaltsev**
## Purpose

Remove results display in case of there are not any search queries or filters in the scope of the [story](https://issues.folio.org/browse/UIIN-79)

## Screenshots

inventory_no_results_by_default

### CODE REVIEW

File: src/Instances.js [hunk 1]
```diff
@@ -92,61 +95,38 @@ class Instances extends React.Component {
...
+              const identifierType = resourceData.identifier_types.records.find(type => type.name.toLowerCase() === queryIndex);
+              const identifierTypeId = identifierType ? identifierType.id : 'identifier-type-not-found';
+              queryTemplate = searchableIndex.makeQuery(identifierTypeId);
+            } else {
+              queryTemplate = searchableIndex.makeQuery();
             }
 
-            return cql;
-          },
+            resourceData.query = { ...resourceData.query, ...{ qindex: '' } };
```
Comment (SergiyVSergiyenko) [inline src/Instances.js]: What about 

> resourceData.query = { ...resourceData.query, qindex: '' };
?
As for me it looks easier.

File: src/Instances.js [hunk 2]
```diff
@@ -92,61 +95,38 @@ class Instances extends React.Component {
...
+              resourceData,
+              logger
+            ] = args;
+            const queryIndex = resourceData.query.qindex ? resourceData.query.qindex : 'all';
+            const searchableIndex = searchableIndexes.find(idx => idx.value === queryIndex);
+            let queryTemplate = '';
+
+            if (queryIndex === 'isbn' || queryIndex === 'issn') {
+              const identifierType = resourceData.identifier_types.records.find(type => type.name.toLowerCase() === queryIndex);
+              const identifierTypeId = identifierType ? identifierType.id : 'identifier-type-not-found';
```
Comment (VictorSoroka1) [inline src/Instances.js]: Please add an empty line after variable declaration.

File: src/Instances.js [hunk 3]
```diff
@@ -92,61 +95,38 @@ class Instances extends React.Component {
...
+              queryTemplate = searchableIndex.makeQuery(identifierTypeId);
```
[the remark this answers is not in the document]
Comment (YevheniiMaltsev) [reply src/Instances.js]: Yes, you are right. Thanks!

File: src/Instances.js (hunk 2 above)
Comment (YevheniiMaltsev) [reply src/Instances.js]: Fixed

File: src/Instances.js (hunk 1 above)
Comment (YevheniiMaltsev) [reply src/Instances.js]: Thanks. Fixed

**Comment by YevheniiMaltsev** [comment]
@zburke Thanks for your comment. I agree with you and this is a good option to add the attached solution to solve the issue. And it was agreed to use makeQueryFunction to remove some code duplication and make the code a little bit simpler. It looks like this refactoring worth it because it is easier to support makeQueryFunction instead of partly copied code in every module. 
And I believe that we will be able to make the code base even better in the future.

**Pull request merged**